esmvaltool/recipes/recipe_OceanPhysics.yml

```diff
@@ -0,0 +1,451 @@
+
+  #####
+  # Data with Errors right now -June 2018
+  ###
+  # Problem with times
+#  - {dataset: MIROC-ESM, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+#  - {dataset: MIROC-ESM-CHEM, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+
+  # Unstructured grids
+#  - {dataset: MPI-ESM-LR, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2003}
+#  - {dataset: MPI-ESM-MR, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+#  - {dataset: ACCESS1-0, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+#  - {dataset: ACCESS1-3, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 2001, end_year: 2004}
+
+
+preprocessors:
+  # --------------------------------------------------
+  # Time series preprocessors
+  # --------------------------------------------------
+  prep_timeseries_1: # For 2D fields
+    average_region:
+      coord1: longitude
+      coord2: latitude
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  prep_timeseries_2: # For specific levels of 3D fields
+    fix_data:
+    fix_metadata:
+    extract_levels:
+      levels: [0., 10., 100., 1000.]
+      scheme: linear_horizontal_extrapolate_vertical
+    average_region:
+      coord1: longitude
+      coord2: latitude
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  prep_timeseries_3: # For regional fields
+    extract_region:
+      start_longitude: -80.
+      end_longitude: 30.
+      start_latitude: -80.
+      end_latitude: 80.
+    average_region:
+      coord1: longitude
+      coord2: latitude
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  prep_timeseries_4: # For regional fields at depth
+    extract_levels:
+      levels: [0., 10., 100., 1000.]
+      scheme: linear_horizontal_extrapolate_vertical
+    extract_region:
+      start_longitude: -80.
+      end_longitude: 30.
+      start_latitude: -80.
+      end_latitude: 80.
+    average_region:
+      coord1: longitude
+      coord2: latitude
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  prep_timeseries_5: # For Global Volume Averaged
+    average_volume:
+      coord1: longitude
+      coord2: latitude
+      coordz: depth
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  prep_timeseries_6: # For regional volume averaged
+    extract_region:
+      start_longitude: -80.
+      end_longitude: 30.
+      start_latitude: -80.
+      end_latitude: 80.
+    average_volume:
+      coord1: longitude
+      coord2: latitude
+      coordz: depth
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  prep_timeseries_7: # For regional volume averaged (at the surface)
+    fix_data:
```

Comment:
And here ..

Reply:
Do you need the empty `fix_data` and `fix_metadata` dicts? Fixes should be applied fully automatically by the preprocessor and there should be no need to specify them here.